pkg/apis/networking/v1/defaults_test.go

```diff
@@ -234,6 +237,99 @@ func TestSetDefaultNetworkPolicy(t *testing.T) {
 	}
 }
 
+func TestSetDefaultsForIngressClassParametersReference(t *testing.T) {
+	tests := []struct {
+		original          *networkingv1.IngressClass
+		expected          *networkingv1.IngressClass
+		enableFeatureGate bool
```

Comment:
Nit: Tests like this sometimes end up testing multiple feature gates. In case that happens here, it would be helpful to have a more specific name like `enableNamespaceScopedParams`.